src/main/java/com/netgrif/application/engine/petrinet/service/PetriNetService.java

```diff
@@ -377,7 +377,7 @@ public Page<PetriNetReference> search(Map<String, Object> criteria, LoggedUser u
         Query query = new Query();
         Query query_total = new Query();
 
-        if (!user.isAdmin())
+        if (user.getAuthorities().stream().noneMatch(a -> a.getAuthority().equals("PROCESS_VIEW_ALL")))
```

Comment:
A constant should be used instead of a string

Reply:
Resolved